src/components/application_manager/src/commands/mobile/on_vehicle_data_notification.cc

```diff
@@ -56,13 +56,24 @@ void OnVehicleDataNotification::Run() {
   std::vector<smart_objects::SmartObject> appSO;
 
   const VehicleData& vehicle_data = MessageHelper::vehicle_data();
+  const smart_objects::SmartObject incoming_msg_params = (*message_)[strings::msg_params];
   VehicleData::const_iterator it = vehicle_data.begin();
+  std::string key;
 
   for (; vehicle_data.end() != it; ++it) {
```

Comment:
@KVGrygoriev 
since this part of code should be modified better to use range based for loop
for (const auto element : vehicle_data)

Reply:
cycle didn't was modify